src/sentry/constants.py

```diff
@@ -189,6 +189,8 @@ def get_all_languages():
 
 OK_PLUGIN_SAVED = _('Configuration for the {name} integration has been saved.')
 
+WARN_SESSION_EXPIRED = 'Your session has expired.'  # TODO: translate this
```

Comment:
I can try, but I don' tknow what the error is. I can just try to fix it after it's merged.